source/documentation/deploying_services/opensearch.md

```diff
@@ -0,0 +1,281 @@
+
+    ```
+    cf service SERVICE_NAME
+    ```
+
+    for example:
+
+    ```
+    cf service my-opensearch-service
+    ```
+
+    When `cf service SERVICE_NAME` returns a `create succeeded` status, you have set up the service instance. An example output could be:
+
+    ```
+    name:            my-opensearch-service
+    service:         opensearch
+    tags:
+    plan:            small-ha-1
+    description:     OpenSearch instances provisioned via Aiven
+    documentation:   https://docs.cloud.service.gov.uk/deploying_services/opensearch/
+    dashboard:        
+    service broker:  aiven-broker
+
+    There are no bound apps for this service.
+
+    Showing status of last operation from service my-opensearch-service...
+
+    status:    create succeeded
+    message:   Last operation succeeded
+    started:   2018-08-02T10:17:30Z
+    updated:   2018-08-02T10:21:35Z
+    ```
+
+### Bind an OpenSearch service to your apps
+
+To access the cache from the app, you must bind your app to the OpenSearch service.
+
+1. Use the [app's manifest](/deploying_apps.html#deploying-public-apps) to bind the app to the service instance. It will bind automatically when you next deploy your app. An example manifest:
+
+    ```
+    --
+    applications:
+      - name: my-app
+        services:
+        - my-opensearch-service
+    ```
+
+1. Deploy your app in line with your normal deployment process.
+
+This binds your app to a service instance called `my-opensearch-service`.
+
+Refer to the [Cloud Foundry documentation on deploying with app manifests](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#services-block) for more information.
+
+#### Use the cf bind-service command
+
+Alternatively, you can manually bind your service instance to your app.
+
+1. Run the following:
+
+    ```
+    cf bind-service APP_NAME SERVICE_NAME
+    ```
+
+    where `APP_NAME` is the exact name of a deployed instance of your application and `SERVICE_NAME` is the name of the service instance you created. For example:
+
+    ```
+    cf bind-service my-app my-opensearch-service
+    ```
+
+1. Deploy your app in line with your normal deployment process.
+
+Refer to the [Cloud Foundry documentation on deploying with app manifests](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#services-block) for more information.
+
+<h2 id="amend-the-service">Amend the service</h2>
+
+### Upgrade OpenSearch service plan
+
+You can upgrade your plan using the `cf update-service` command. Run the following in the command line:
+
+```sh
+cf update-service SERVICE_NAME -p NEW_PLAN_NAME
+```
+
+where `SERVICE_NAME` is a unique descriptive name for this service instance, and `NEW_PLAN_NAME` is the name of your new plan. For example:
+
+```sh
+cf update-service my-opensearch-service -p medium-ha-1
+```
+
+The plan upgrade will start immediately and finish within an hour. You can check the status of the upgrade by running `cf services`.
```

Comment:
Should we be a bit less definitive here? Large plans could take more than an hour @cmcnallygds

Reply:
Standard response here of "this is a copypasta from ES docs, are we fixing it individually here and leaving the ES docs bad and dangly?"